test/test_extended_models.py

```diff
@@ -196,24 +212,49 @@ def test_schema_meta_validation(model_fn):
         if missing_fields or unsupported_fields:
             problematic_weights[w] = {"missing": missing_fields, "unsupported": unsupported_fields}
         if w == weights_enum.DEFAULT:
+
             if module_name == "quantization":
                 # parameters() count doesn't work well with quantization, so we check against the non-quantized
                 unquantized_w = w.meta.get("unquantized")
                 if unquantized_w is not None and w.meta.get("num_params") != unquantized_w.meta.get("num_params"):
                     incorrect_params.append(w)
+
+                # the methodology for quantized ops count doesn't work as well, so we take unquantized FLOPs instead
+                calculated_ops = get_ops(model=None, module_name="models", model_name=model_name, weight=unquantized_w)
             else:
-                if w.meta.get("num_params") != sum(p.numel() for p in model_fn(weights=w).parameters()):
+
+                # loading the model and using it for parameter and ops verification
+                kwargs = {"quantize": True} if module_name == "quantization" else {}
+                model = model_fn(weights=w, **kwargs)
+
+                if w.meta.get("num_params") != sum(p.numel() for p in model.parameters()):
                     incorrect_params.append(w)
+
+                kwargs = {}
+                if model_name in detection_models_input_dims:
+                    # detection models have non default height and width
+                    height, width = detection_models_input_dims[model_name]
+                    kwargs = {"h": height, "w": width}
+                calculated_ops = get_ops(
+                    model=model, module_name=module_name, model_name=model_name, weight=w, **kwargs
+                )
+
+            # assert that weight flops are correctly pasted to metadata
+            assert calculated_ops == w.meta["_ops"]
```

Comment:
We shouldn't assert like this because it will fail immediately the test without showing us other issues. Instead we should be collecting all issues in one list and showing them to the user. Previously we had `incorrect_params` which was monitoring issues with the number of parameters. Now that we have more, it's worth switching this into something like `incorrect_meta` and append to it not only the weight but also the meta name that failed. For example: `incorrect_params.append((w, "num_params"))`.